Runner/Program.cs

```diff
@@ -51,7 +53,7 @@ void scanner(IEnumerable<string> repositories, Options options)
                     var results = repositories.Select(repo =>
                     {
                         Thread.Sleep(TimeSpan.FromSeconds(3));
-                        return client.ValidateRepository(githubConfig.Organization, repo).Result;
+                        return client.ValidateRepository(githubConfig.Organization, repo, options.IgnoreRepositoryRules).Result;
```

Comment:
`.ConfigureAwait(false).GetAwaiter().GetResult()` instead of `.Result` is the recommended way to get task results in synchronous code nowadays. But you could consider converting this to use async-awaits.

Reply:
That's a good point. I'll change these to `.ConfigureAwait(false).GetAwaiter().GetResult()`

I'll probably change these to `async` later, because I need to verify that those reports are done in order (or at least that the sleep is done between and not accidentally made parallel)

`CommandLineParser` also had some problems with async, or I just didn't get the syntax right.